database/chefs.js

```diff
@@ -0,0 +1,25 @@
+const db = require('./database.js');
+
```

Comment:
how are you determining what goes in database/chef.js and what stays in the server? should duke and I be doing something similar?

Reply:
I was putting all non-schema helper functions in there; I figured that you and Duke would want to do similarly. ForeverPaws (HRR32 repo) has an example of code organization I've taken some inspiration from: https://github.com/codydaig/Forever-Paws/blob/master/client/src/app/routes/routes.js